doc/examples/segmentation/plot_chanvese.py

```diff
@@ -0,0 +1,25 @@
+""" This is an example to illustrate the us of the Chan-Vese Algorithm.
+"""
+
+from skimage.data import camera
+from skimage.segmentation.chanvese import chan_vese
+import numpy as np
+import matplotlib.pyplot as plt
+
+def _cv_main():
+    U0 = camera().astype('float')/255.
+    print np.max(U0)
+    # Feel free to play around with these parameters to see how they impact the
+    # result
+    cv = chan_vese(U0, mu=0.5, lambda1=1, lambda2=0.5, tol=1, maxiter=100, dt=100)
+    print ("Chan-Vese algorithm finished after "+str(len(cv[1]))+" iterations.")
+    print cv[1]
+    plt.imshow(cv[0])
+    plt.colorbar()
+    plt.show()
+    plt.plot(cv[1])
+    plt.show()
+    return
+    
+if __name__ == '__main__':
```

Comment:
Please, remove this and refactor example to be an "ordinary" script.
See http://scikit-image.org/docs/dev/auto_examples/features_detection/plot_template.html#example-features-detection-plot-template-py for example of this and all points mentioned above.